codespell_lib/data/dictionary.txt

```diff
@@ -860,7 +863,8 @@ afficionados->aficionados
 affilate->affiliate
 affilates->affiliates
 affilliate->affiliate
-affinitied->affinitized
+affinitied->affinities
+affinitized->affinities, affinity,
 affinitze->affinitize
```

Comment:
Hmmm, if we're offering `affinitize` as a correction, does it make sense to report `affinitized` as an error?
https://www.google.com/search?q=affinitize
:shrug:

Reply:
Possibly not then. -ed is more popular in Google Books than -e too! http://app.aspell.net/lookup?dict=en_US-large&words=affinitize

Reply:
I couldn't find affinitize in an english dictionary. and my german dictionaries/deepl have some really strange suggestions.

So let it in or throw it out?

Reply:
We should probably allow affinitized too really, or at least not correct it.